Jenkinsfile

```diff
@@ -4,11 +4,14 @@
 pipeline {
   agent any
   environment {
-    BASE_DIR = "src/github.com/elastic/apm-server"
+    REPO = 'apm-server'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
     NOTIFY_TO = credentials('notify-to')
     JOB_GCS_BUCKET = credentials('gcs-bucket')
     JOB_GCS_CREDENTIALS = 'apm-ci-gcs-plugin'
     CODECOV_SECRET = 'secret/apm-team/ci/apm-server-codecov'
+    GITHUB_CHECK_ITS_NAME = 'Integration Tests'
+    ITS_PIPELINE = 'apm-integration-tests-mbp/PR-470'
```

Comment:
This should be changed as 'apm-integration-tests-mbp/master' as soon as it's merged https://github.com/elastic/apm-integration-testing/pull/470